azurelinuxagent/ga/collect_logs.py

```diff
@@ -121,7 +120,7 @@ def init_protocols(self):
 
     def daemon(self, init_data=False):
         try:
-            if not self._log_collection_allowed():
+            if not is_log_collection_allowed():
```

Comment:
This check is redundant now, not really needed

Reply:
Yup, you're right! Will remove it.